python/ray/serve/_private/replica.py

```diff
@@ -129,6 +129,9 @@ async def __init__(
                 controller_name, namespace=SERVE_NAMESPACE
             )
 
+            # Indicates whether the replica has finished initializing.
+            self.init_finish_event = asyncio.Event()
```

Comment:
nit: use `_` prefix for private variables
```suggestion
            self._init_finish_event = asyncio.Event()
```